src/index.html

```diff
@@ -1,57 +1,54 @@
 <!DOCTYPE html>
 <html>
-
 <head>
   <title>Activity Tracker</title>
   <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700&display=swap" rel="stylesheet">
   <link href="https://fonts.googleapis.com/css?family=Raleway:400,300,600,800,900" rel="stylesheet" type="text/css">
   <script src="https://cdn.jsdelivr.net/npm/chart.js@2.8.0"></script>
   <script src="https://cdn.rawgit.com/kimmobrunfeldt/progressbar.js/0.7.0/dist/progressbar.js"></script>
-
 </head>
-
 <body>
   <section class="welcome-section">
```

Comment:
A header element might be a little more semantic here rather than a section tag

Reply:
Agreed - there is a place that <article> is being used in a way that <section> would be more semantic as well.